specification.yaml

```diff
@@ -94,6 +94,72 @@
         responses:
           204:
             description: No content
+    /timeevents:
```

Comment:
naming-kommentar: Eg ser at lenger nede har vi kalla denne ressursen for "TimeEvent", som eg synes er fint. Det betyr, ihht navnekonvensjoner, at `timeevent` -> `time-event` i url, og `timeevent` -> `time_event` i python. Klassene `TimeeventsAdapter` etc blir då `TimeEventsAdapter`.